docs/getting-started/installation.md

```diff
@@ -146,3 +143,137 @@ When using a different setup, all you have to do is modify the parameters in the
 
     To forward every command to the remote server, use the `:remote console` command.
     Further documentation can be found in the [TinkerPop reference docs](https://tinkerpop.apache.org/docs/{{ tinkerpop_version }}/reference/#console-remote-console)
+
+## Using the Pre-Packaged Distribution
```

Comment:
I think this section should be merged somehow with the section _Local Installation_ above as this section also explains how to start JanusGraph Server and connect to it from the console. Although that could also be done in a follow-up PR to make the review easier (or at least in separate commits).